go.mod

```diff
@@ -1,31 +1,37 @@
-module github.com/metrumresearchgroup/babylon
+module bbi
 
-go 1.13
+go 1.15
 
 require (
+	github.com/apcera/termtables v0.0.0-20170405184538-bcbc5dc54055 // indirect
 	github.com/clbanning/mxj v1.8.4
-	github.com/coreos/bbolt v1.3.3
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-chi/chi v4.0.2+incompatible
-	github.com/go-chi/render v1.0.1
 	github.com/gobwas/glob v0.2.3
-	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.3.2
-	github.com/google/uuid v1.1.1
-	github.com/kr/pretty v0.1.0
-	github.com/logrusorgru/aurora v0.0.0-20191116043053-66b7ad493a23
+	github.com/google/uuid v1.1.2
+	github.com/kr/pretty v0.2.1
+	github.com/kr/text v0.2.0 // indirect
+	github.com/logrusorgru/aurora v2.0.3+incompatible
+	github.com/magiconair/properties v1.8.4 // indirect
+	github.com/mattn/go-runewidth v0.0.9 // indirect
+	github.com/metrumresearchgroup/bbi v0.2.0
```

Comment:
Hmm, bbi requires an earlier version of itself?

Reply:
I'm not at all confident, but poking around at a few other repos, I wondering if things should look like this (builds for me):

<details>
<summary>diff</summary>

```diff
diff --git a/cmd/bbi/main.go b/cmd/bbi/main.go
index 4e2d134..736d2e1 100644
--- a/cmd/bbi/main.go
+++ b/cmd/bbi/main.go
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"bbi/cmd"
+	"github.com/metrumresearchgroup/bbi/cmd"
 )
 
 // buildTime  can be set from LDFLAGS during development
diff --git a/cmd/clean.go b/cmd/clean.go
index 3d37b2f..9763075 100644
--- a/cmd/clean.go
+++ b/cmd/clean.go
@@ -20,8 +20,8 @@ import (
 	"path/filepath"
 	"strings"
 
-	"bbi/runner"
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/runner"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
diff --git a/cmd/covcor.go b/cmd/covcor.go
index 47f3baa..b448973 100644
--- a/cmd/covcor.go
+++ b/cmd/covcor.go
@@ -15,7 +15,7 @@
 package cmd
 
 import (
-	parser "bbi/parsers/nmparser"
+	parser "github.com/metrumresearchgroup/bbi/parsers/nmparser"
 	"encoding/json"
 	"fmt"
 	log "github.com/sirupsen/logrus"
diff --git a/cmd/init.go b/cmd/init.go
index 3f87b66..e7b956f 100644
--- a/cmd/init.go
+++ b/cmd/init.go
@@ -7,7 +7,7 @@ import (
 	"regexp"
 	"strings"
 
-	"bbi/configlib"
+	"github.com/metrumresearchgroup/bbi/configlib"
 	"github.com/ghodss/yaml"
 
 	"github.com/spf13/afero"
diff --git a/cmd/local.go b/cmd/local.go
index ce15e9f..c1b4247 100644
--- a/cmd/local.go
+++ b/cmd/local.go
@@ -1,7 +1,7 @@
 package cmd
 
 import (
-	"bbi/configlib"
+	"github.com/metrumresearchgroup/bbi/configlib"
 	"crypto/md5"
 	"encoding/json"
 	"errors"
@@ -16,8 +16,8 @@ import (
 
 	"os"
 
-	"bbi/runner"
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/runner"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/metrumresearchgroup/turnstile"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
diff --git a/cmd/nonmem.go b/cmd/nonmem.go
index e421bd9..3e475d6 100644
--- a/cmd/nonmem.go
+++ b/cmd/nonmem.go
@@ -29,10 +29,10 @@ import (
 	"text/template"
 	"time"
 
-	"bbi/configlib"
-	parser "bbi/parsers/nmparser"
-	"bbi/runner"
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/configlib"
+	parser "github.com/metrumresearchgroup/bbi/parsers/nmparser"
+	"github.com/metrumresearchgroup/bbi/runner"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/metrumresearchgroup/turnstile"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/afero"
diff --git a/cmd/nonmem_test.go b/cmd/nonmem_test.go
index 6dcd9e8..3a4b7dd 100644
--- a/cmd/nonmem_test.go
+++ b/cmd/nonmem_test.go
@@ -7,7 +7,7 @@ import (
 	"reflect"
 	"testing"
 
-	"bbi/configlib"
+	"github.com/metrumresearchgroup/bbi/configlib"
 	"github.com/google/uuid"
 	"github.com/spf13/afero"
 )
diff --git a/cmd/params.go b/cmd/params.go
index b7cf77f..fa00e14 100644
--- a/cmd/params.go
+++ b/cmd/params.go
@@ -17,8 +17,8 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
-	parser "bbi/parsers/nmparser"
-	"bbi/utils"
+	parser "github.com/metrumresearchgroup/bbi/parsers/nmparser"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/scylladb/go-set/strset"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
diff --git a/cmd/project.go b/cmd/project.go
index e4409ff..a606540 100644
--- a/cmd/project.go
+++ b/cmd/project.go
@@ -25,8 +25,8 @@ import (
 
 	"strings"
 
-	parser "bbi/parsers/nmparser"
-	"bbi/utils"
+	parser "github.com/metrumresearchgroup/bbi/parsers/nmparser"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
diff --git a/cmd/reclean.go b/cmd/reclean.go
index cc39fd1..ef45c44 100644
--- a/cmd/reclean.go
+++ b/cmd/reclean.go
@@ -17,7 +17,7 @@ package cmd
 import (
 	"fmt"
 
-	"bbi/runner"
+	"github.com/metrumresearchgroup/bbi/runner"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
diff --git a/cmd/root.go b/cmd/root.go
index 54a3c25..a5f816e 100644
--- a/cmd/root.go
+++ b/cmd/root.go
@@ -15,7 +15,7 @@
 package cmd
 
 import (
-	"bbi/configlib"
+	"github.com/metrumresearchgroup/bbi/configlib"
 	"fmt"
 	"github.com/metrumresearchgroup/turnstile"
 	log "github.com/sirupsen/logrus"
diff --git a/cmd/run.go b/cmd/run.go
index a1096a6..99521ab 100644
--- a/cmd/run.go
+++ b/cmd/run.go
@@ -1,7 +1,7 @@
 package cmd
 
 import (
-	"bbi/configlib"
+	"github.com/metrumresearchgroup/bbi/configlib"
 	"bytes"
 	"github.com/metrumresearchgroup/turnstile"
 	log "github.com/sirupsen/logrus"
diff --git a/cmd/scaffold.go b/cmd/scaffold.go
index af2a6bb..3a68ed1 100644
--- a/cmd/scaffold.go
+++ b/cmd/scaffold.go
@@ -19,7 +19,7 @@ import (
 	"log"
 	"path/filepath"
 
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
diff --git a/cmd/sge.go b/cmd/sge.go
index df939db..c1f9419 100644
--- a/cmd/sge.go
+++ b/cmd/sge.go
@@ -1,7 +1,7 @@
 package cmd
 
 import (
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"bytes"
 	"errors"
 	log "github.com/sirupsen/logrus"
@@ -15,7 +15,7 @@ import (
 
 	"os"
 
-	"bbi/configlib"
+	"github.com/metrumresearchgroup/bbi/configlib"
 	"github.com/metrumresearchgroup/turnstile"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
diff --git a/cmd/sge_test.go b/cmd/sge_test.go
index ce4e215..27f4f24 100644
--- a/cmd/sge_test.go
+++ b/cmd/sge_test.go
@@ -1,7 +1,7 @@
 package cmd
 
 import (
-	"bbi/configlib"
+	"github.com/metrumresearchgroup/bbi/configlib"
 	"testing"
 )
 
diff --git a/cmd/summary.go b/cmd/summary.go
index 98355f8..5d69389 100644
--- a/cmd/summary.go
+++ b/cmd/summary.go
@@ -15,7 +15,7 @@
 package cmd
 
 import (
-	parser "bbi/parsers/nmparser"
+	parser "github.com/metrumresearchgroup/bbi/parsers/nmparser"
 	"encoding/json"
 	"fmt"
 	log "github.com/sirupsen/logrus"
diff --git a/go.mod b/go.mod
index 0cabbd9..db8d372 100644
--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module bbi
+module github.com/metrumresearchgroup/bbi
 
 go 1.15
 
@@ -14,7 +14,6 @@ require (
 	github.com/logrusorgru/aurora v2.0.3+incompatible
 	github.com/magiconair/properties v1.8.4 // indirect
 	github.com/mattn/go-runewidth v0.0.9 // indirect
-	github.com/metrumresearchgroup/bbi v0.2.0
 	github.com/metrumresearchgroup/turnstile v0.1.0
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/mapstructure v1.4.0 // indirect
diff --git a/parsers/nmparser/get_model_output.go b/parsers/nmparser/get_model_output.go
index a3ff6fe..db434b8 100644
--- a/parsers/nmparser/get_model_output.go
+++ b/parsers/nmparser/get_model_output.go
@@ -7,7 +7,7 @@ import (
 	"strconv"
 	"strings"
 
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/afero"
 )
diff --git a/parsers/nmparser/parse_lst_file.go b/parsers/nmparser/parse_lst_file.go
index e87fa84..87feb8f 100644
--- a/parsers/nmparser/parse_lst_file.go
+++ b/parsers/nmparser/parse_lst_file.go
@@ -1,7 +1,7 @@
 package parser
 
 import (
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"fmt"
 	"math"
 	"sort"
diff --git a/parsers/nmparser/read_grd.go b/parsers/nmparser/read_grd.go
index 671d559..3581f51 100644
--- a/parsers/nmparser/read_grd.go
+++ b/parsers/nmparser/read_grd.go
@@ -4,7 +4,7 @@ import (
 	"strconv"
 	"strings"
 
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 )
 
 // ParseGrdLines parses out the ext lines into a data structure for final parameter processing.
diff --git a/parsers/nmparser/read_grd_test.go b/parsers/nmparser/read_grd_test.go
index 073ea12..c312152 100644
--- a/parsers/nmparser/read_grd_test.go
+++ b/parsers/nmparser/read_grd_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 	"testing"
 
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/stretchr/testify/assert"
 )
 
diff --git a/runner/clean_and_copy_est.go b/runner/clean_and_copy_est.go
index 440b8c1..e5fa8ff 100644
--- a/runner/clean_and_copy_est.go
+++ b/runner/clean_and_copy_est.go
@@ -8,7 +8,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/spf13/afero"
 )
 
diff --git a/runner/estimate_model.go b/runner/estimate_model.go
index 034e6fc..542d4dc 100644
--- a/runner/estimate_model.go
+++ b/runner/estimate_model.go
@@ -8,7 +8,7 @@ import (
 
 	"errors"
 
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/spf13/afero"
 )
 
diff --git a/runner/prepare_est_run.go b/runner/prepare_est_run.go
index dbc7266..2a04873 100644
--- a/runner/prepare_est_run.go
+++ b/runner/prepare_est_run.go
@@ -5,8 +5,8 @@ import (
 
 	"fmt"
 
-	parser "bbi/parsers/nmparser"
-	"bbi/utils"
+	parser "github.com/metrumresearchgroup/bbi/parsers/nmparser"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/spf13/afero"
 )
 
diff --git a/runner/prepare_to_execute.go b/runner/prepare_to_execute.go
index 24deea5..052bf12 100644
--- a/runner/prepare_to_execute.go
+++ b/runner/prepare_to_execute.go
@@ -1,7 +1,7 @@
 package runner
 
 import "strings"
-import parser "bbi/parsers/nmparser"
+import parser "github.com/metrumresearchgroup/bbi/parsers/nmparser"
 
 //PrepareForExecution parses and prepares strings from a file for execution in a different context
 // for example, replacing the $DATA path
diff --git a/runner/run_model.go b/runner/run_model.go
index 7f39df0..8d6d01e 100644
--- a/runner/run_model.go
+++ b/runner/run_model.go
@@ -9,7 +9,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	"bbi/utils"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/spf13/afero"
 )
 
diff --git a/scratch/main.go b/scratch/main.go
index 0500637..a4106cb 100644
--- a/scratch/main.go
+++ b/scratch/main.go
@@ -4,8 +4,8 @@ import (
 	"fmt"
 	"time"
 
-	parser "bbi/parsers/nmparser"
-	"bbi/utils"
+	parser "github.com/metrumresearchgroup/bbi/parsers/nmparser"
+	"github.com/metrumresearchgroup/bbi/utils"
 	"github.com/kr/pretty"
 	"github.com/mitchellh/go-homedir"
 )
```

</details>

Reply:
The module line with the full path to bbi means the require line is no longer needed.

It was probably added by someone importing the full path to bbi in a file, and go mod fixing it up as an import. Weird, but possible.